gm4_soul_glass/data/gm4_soul_glass/tags/blocks/beacon_passable.json

```diff
@@ -1,83 +1,33 @@
 {
     "values": [
-        "minecraft:air",
-        "minecraft:cave_air",
-        "minecraft:void_air",
+        "#gm4:air",
```

Comment:
missing #minecraft:campfires (replace minecraft:campfire), chain, composter, soul_lantern, soul_torch, torch, wall_soul_torch, wall_torch

Reply:
This isn't part of the base and therefore doesn't belong here, or does it?

Reply:
Yes and no. While this isn't part of the base and would need these changes anyway apart from the base update, the base update alphabetizes block tags and also changes block tags to utilize the base block tags wherever possible.